packages/sources/amberdata/src/endpoint/crypto.ts

```diff
@@ -0,0 +1,64 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
```

Comment:
Looks like this was `price.ts` re-named, but isn't being used.

I'd clean up one or the other.

Reply:
This gets cleaned up in the `price-endpoint-refactor` that uses this PR as a base